src/main/java/com/evolveum/polygon/connector/msgraphapi/UserProcessing.java

```diff
@@ -376,6 +387,9 @@ protected ObjectClassInfo objectClassInfo() {
         ;
         userObjClassBuilder.addAttributeInfo(attrInterests.build());
 
+        AttributeInfoBuilder userPhoto = new AttributeInfoBuilder(ATTR_USERPHOTO);
```

Comment:
Please add the setReturnedByDefault(false) property for such an attribute, the attribute might not be needed in most deployments of the connector and additionally, it requires an additional roundtrip to the remote service.

Reply:
Also, please add the conditionals needed regarding the retrieval of the property which check "ATTRS_TO_GET" operation options in the correct places later in code.

Reply:
both done